src/cfnlint/data/AdditionalSpecs/BasedOnValue.json

```diff
@@ -47,6 +47,22 @@
                     ]
                 }
             ]
+        },
+        "AWS::RDS::DBInstance": {
+            "Engine": [
+                {
+                    "Regex": "(aurora|aurora-mysql|aurora-postgresql)",
+                    "UnwantedProperties": [
+                        "AllocatedStorage",
+                        "BackupRetentionPeriod",
+                        "CopyTagsToSnapshot",
```

Comment:
[linking discussion on whether this property should be dropped from this list](https://github.com/aws-cloudformation/cfn-python-lint/pull/1409#issuecomment-601683080)

Reply:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1409